src/shared/components/v2/iconsRenewal/BtnIcon.tsx

```
@@ -0,0 +1,64 @@
+  return (
+    <button
+      className={styles.btnIcon({
+        status: isPressed ? 'PRESSED' : 'DEFAULT',
+        disabled,
+      })}
+      disabled={disabled}
+      onClick={onClick}
+      onPointerDown={onPointerDown}
+      onPointerUp={onPointerUp}
+      onPointerLeave={onPointerLeave}
+    >
```

Biome (2.4.7)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)